code/modules/projectiles/guns/energy/magic.dm

```diff
@@ -313,6 +347,11 @@ obj/item/weapon/gun/energy/staff/focus/attack_self(mob/living/user as mob)
 	max_shots = 10
 	projectile_type = /obj/item/projectile/forcebolt
 
+/obj/item/weapon/gun/energy/wand/force/pickup(mob/living/user as mob)
+	if(!user.is_wizard())
+		user << "<span class='warning'>When you pick up \the [src], it makes your arm thump unpleasantly. You don't think you should try and use this...</span>"

```

Comment:
Along with this being the 10th one, what the hell does "it makes your arm thump" mean? Makes your arm hit something?